<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RSTP Crafter</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../Desktop/Beer-icon.png</normaloff>../../../Desktop/Beer-icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>41</x>
      <y>210</y>
      <width>39</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TCN ack</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>86</x>
      <y>210</y>
      <width>29</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Agree</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>126</x>
      <y>210</y>
      <width>28</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Frwrd</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>167</x>
      <y>210</y>
      <width>27</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Learn</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>210</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Port Roles</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>210</y>
      <width>22</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Prop</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>210</y>
      <width>20</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TCN</string>
    </property>
   </widget>
   <widget class="QWidget" name="parentWidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>481</width>
      <height>131</height>
     </rect>
    </property>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>21</width>
       <height>100</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>c2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_18">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>00</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_34">
        <property name="text">
         <string>00</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_50">
        <property name="text">
         <string>0f</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_10">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>10</y>
       <width>21</width>
       <height>100</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_12">
      <item>
       <widget class="QLineEdit" name="lineEdit_10">
        <property name="text">
         <string>91</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_26">
        <property name="text">
         <string>00</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_42">
        <property name="text">
         <string>80</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_58">
        <property name="text">
         <string>00</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_11">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>10</y>
       <width>21</width>
       <height>100</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_13">
      <item>
       <widget class="QLineEdit" name="lineEdit_11">
        <property name="text">
         <string>bf</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_27">
        <property name="text">
         <string>00</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_43">
        <property name="text">
         <string>03</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_59">
        <property name="text">
         <string>00</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_9">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>10</y>
       <width>21</width>
       <height>100</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_11">
      <item>
       <widget class="QLineEdit" name="lineEdit_9">
        <property name="text">
         <string>0c</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_25">
        <property name="text">
         <string>00</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_41">
        <property name="text">
         <string>00</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_57">
        <property name="text">
         <string>00</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_6">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>10</y>
       <width>21</width>
       <height>100</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <widget class="QLineEdit" name="lineEdit_6">
        <property name="text">
         <string>90</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_22">
        <property name="toolTip">
         <string/>
        </property>
        <property name="toolTipDuration">
         <number>1</number>
        </property>
        <property name="whatsThis">
         <string comment="Root bridge priority"/>
        </property>
        <property name="text">
         <string>80</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_38">
        <property name="text">
         <string>00</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_54">
        <property name="text">
         <string>00</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_13">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>10</y>
       <width>21</width>
       <height>74</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_15">
      <item>
       <widget class="QLineEdit" name="lineEdit_13">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>27</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_29">
        <property name="text">
         <string>00</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_45">
        <property name="text">
         <string>00</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_5">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>9</y>
       <width>21</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QLineEdit" name="lineEdit_5">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>00</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_21">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>00</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_37">
        <property name="text">
         <string>00</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_53">
        <property name="text">
         <string>00</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>21</width>
       <height>100</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="lineEdit_0">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>01</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_16">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>03</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_32">
        <property name="text">
         <string>00</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_48">
        <property name="text">
         <string>02</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_3">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>21</width>
       <height>100</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>00</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_19">
        <property name="text">
         <string>02</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_35">
        <property name="text">
         <string>00</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_51">
        <property name="text">
         <string>00</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_4">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>21</width>
       <height>100</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QLineEdit" name="lineEdit_4">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>00</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_20">
        <property name="text">
         <string>02</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_36">
        <property name="text">
         <string>00</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_52">
        <property name="text">
         <string>00</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_12">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>10</y>
       <width>21</width>
       <height>70</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_14">
      <item>
       <widget class="QLineEdit" name="lineEdit_12">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>00</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_28">
        <property name="text">
         <string>00</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_44">
        <property name="text">
         <string>00</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_15">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>10</y>
       <width>21</width>
       <height>74</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_17">
      <item>
       <widget class="QLineEdit" name="lineEdit_15">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>42</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_31">
        <property name="text">
         <string>00</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_47">
        <property name="text">
         <string>00</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>21</width>
       <height>100</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLineEdit" name="lineEdit_1">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>80</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_17">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>00</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_33">
        <property name="text">
         <string>00</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_49">
        <property name="text">
         <string>00</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_8">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>10</y>
       <width>21</width>
       <height>100</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_10">
      <item>
       <widget class="QLineEdit" name="lineEdit_8">
        <property name="text">
         <string>ba</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_24">
        <property name="text">
         <string>00</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_40">
        <property name="text">
         <string>00</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_56">
        <property name="text">
         <string>00</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_14">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>10</y>
       <width>21</width>
       <height>70</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_16">
      <item>
       <widget class="QLineEdit" name="lineEdit_14">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>42</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_30">
        <property name="text">
         <string>00</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_46">
        <property name="text">
         <string>14</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_7">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>21</width>
       <height>100</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <widget class="QLineEdit" name="lineEdit_7">
        <property name="text">
         <string>e2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_23">
        <property name="text">
         <string>00</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_39">
        <property name="text">
         <string>00</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_55">
        <property name="text">
         <string>00</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="flagWidget" native="true">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>231</y>
      <width>507</width>
      <height>41</height>
     </rect>
    </property>
    <widget class="QSpinBox" name="spinBox_5">
     <property name="geometry">
      <rect>
       <x>225</x>
       <y>10</y>
       <width>37</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="sendButton">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>9</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <widget class="QPushButton" name="setFlagsButton">
     <property name="geometry">
      <rect>
       <x>342</x>
       <y>9</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Set</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>9</y>
       <width>25</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Flags</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_0">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maximum">
      <number>1</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_1">
     <property name="geometry">
      <rect>
       <x>77</x>
       <y>10</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maximum">
      <number>1</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_2">
     <property name="geometry">
      <rect>
       <x>114</x>
       <y>10</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maximum">
      <number>1</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_3">
     <property name="geometry">
      <rect>
       <x>151</x>
       <y>10</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maximum">
      <number>1</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_4">
     <property name="geometry">
      <rect>
       <x>188</x>
       <y>10</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maximum">
      <number>1</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_6">
     <property name="geometry">
      <rect>
       <x>268</x>
       <y>10</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maximum">
      <number>1</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_7">
     <property name="geometry">
      <rect>
       <x>305</x>
       <y>10</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maximum">
      <number>1</number>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>84</width>
      <height>44</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Pcap Interface</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>5</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>label</zorder>
   <zorder>spinBox</zorder>
   <zorder>commandLinkButton</zorder>
   <zorder>lineEdit</zorder>
   <zorder>lineEdit_2</zorder>
   <zorder>lineEdit_3</zorder>
   <zorder>lineEdit_4</zorder>
   <zorder>lineEdit_5</zorder>
   <zorder>lineEdit_6</zorder>
   <zorder>lineEdit_7</zorder>
   <zorder>lineEdit_8</zorder>
   <zorder>lineEdit_9</zorder>
   <zorder>lineEdit_0</zorder>
   <zorder>lineEdit_16</zorder>
   <zorder>lineEdit_32</zorder>
   <zorder>lineEdit_48</zorder>
   <zorder>label_2</zorder>
   <zorder>spinBox_7</zorder>
   <zorder>spinBox_6</zorder>
   <zorder>spinBox_2</zorder>
   <zorder>spinBox_1</zorder>
   <zorder>spinBox_3</zorder>
   <zorder>spinBox_4</zorder>
   <zorder>spinBox_0</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
   <zorder>label_8</zorder>
   <zorder>label_9</zorder>
   <zorder>parentWidget</zorder>
   <zorder>label_2</zorder>
   <zorder>flagWidget</zorder>
   <zorder>verticalSpacer</zorder>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
